definitions/npm/lodash_v4.x.x/flow_v0.63.x/lodash_v4.x.x.js

```diff
@@ -158,7 +158,7 @@ declare module "lodash" {
 
   declare type NestedArray<T> = Array<Array<T>>;
 
-  declare type matchesIterateeShorthand = Object;
+  declare type matchesIterateeShorthand = {[id: any]: any};
```

Comment:
Since `Object` now means `any` we can require a very liberal indexed object definition instead.